src/core/util/data_version.pl

```diff
@@ -178,6 +178,10 @@
     [Instance_Object] = Object.instance_objects,
     terminus_store:layer_to_id(Instance_Object.read, Layer_Id_String),
     atom_string(Layer_Id, Layer_Id_String).
+extract_data_version(Descriptor, _Object, data_version(commit, Commit_Id)) :-
```

Comment:
if we are here, I assume we fully managed to open the descriptor?
I'm a little worried since we aren't actually doing anything with the transaction object here. So no check is done here to make sure that the commit_id actually exists. But if we already opened the transaction by this point, that's obviously unnecessary.